Makefile

```
@@ -1,9 +1,21 @@
-.PHONY: up
+.PHONY: up down sparse crawler
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)